src/Symfony/Component/Serializer/Normalizer/ConstraintViolationListNormalizer.php

```diff
@@ -32,8 +34,12 @@ class ConstraintViolationListNormalizer implements NormalizerInterface, Cacheabl
     private $defaultContext;
     private $nameConverter;
 
-    public function __construct($defaultContext = [], NameConverterInterface $nameConverter = null)
+    public function __construct(/** ContextInterface */ $defaultContext = null, NameConverterInterface $nameConverter = null)
```

Comment:
is `null` a good idea ? It does not seems to be allowed before.

Reply:
It should be an `iterable` defaulting to an empty array IMHO